cpp/src/strings/regex/regcomp.cpp

```diff
@@ -570,182 +570,180 @@ class regex_parser {
  * @brief The compiler converts class list into instructions.
  */
 class regex_compiler {
-  reprog& m_prog;
-
-  struct Node {
+  struct and_node {
     int id_first;
     int id_last;
   };
 
-  int cursubid;
-  int pushsubid;
-  std::vector<Node> andstack;
-
-  struct Ator {
+  struct re_operator {
     int t;
     int subid;
   };
 
-  std::vector<Ator> atorstack;
-
-  bool lastwasand;
-  int nbra;
+  reprog& _prog;
```

Comment:
Are these underscores intended to represent "internal" members? If so, given how many of these there are you eventually considering changing them to have private visibility instead of just communicating this by naming convention?

Reply:
These are already private. The underscore helps a human understand their scope compared to local variables in some long functions.

Reply:
Got it, so you intend these similar to the `m_` prefix that a lot of other projects use for members.